### ISSUE TITLE
Wait for file to exist

### ISSUE THREAD

**Issue by Ricardonacif** [jeffbski/wait-on#19]
Hey,

Is there an option to wait for a file to be created? If not, can I PR this?

**Comment by jeffbski** [comment]
Yes that was the original use case, waiting for a file to exist and not be changing. For example one can simply wait for the build.min.js from a build to be complete.

**Comment by Ricardonacif** [comment]
But that's currently not working right?

**Comment by jeffbski** [comment]
Yes, it is still part of the main use cases. We have passing tests that indicate that it still works. 

The CLI is tested for that case here
https://github.com/jeffbski/wait-on/blob/master/test/cli.mocha.js#L57-77

and for the API
https://github.com/jeffbski/wait-on/blob/master/test/api.mocha.js#L46-65

If you are using the CLI then if your path has spaces and such be sure to use quotes around it so the shell can parse it properly. 

If you have an example that doesn't work, I am happy to look at.

**Comment by baconmania** [comment]
Just to clarify, it doesn't seem that "wait for file to exist" is supported. "Wait for file to exist _and the size to be stable_" is supported, but if you just want to watch for a file to come into existence and be immediately notified (even if the file is continually growing), that's not currently supported. Correct me if I'm wrong.
